lightning/src/chain/package.rs

```diff
@@ -959,10 +959,10 @@ impl PackageTemplate {
 	pub(crate) fn get_height_timer(&self, current_height: u32) -> u32 {
 		let mut height_timer = current_height + LOW_FREQUENCY_BUMP_INTERVAL;
 		let timer_for_target_conf = |target_conf| -> u32 {
-			if target_conf <= current_height + MIDDLE_FREQUENCY_BUMP_INTERVAL {
-				current_height + HIGH_FREQUENCY_BUMP_INTERVAL
-			} else if target_conf <= current_height + LOW_FREQUENCY_BUMP_INTERVAL {
+			if target_conf <= current_height + LOW_FREQUENCY_BUMP_INTERVAL {
 				current_height + MIDDLE_FREQUENCY_BUMP_INTERVAL
+			} else if target_conf <= current_height + MIDDLE_FREQUENCY_BUMP_INTERVAL {
+				current_height + HIGH_FREQUENCY_BUMP_INTERVAL
```

Comment:
I'm confused, if `current_height` is 100 and `target_conf` is 101, then we want to return `current_height + HIGH_FREQUENCY_BUMP_INTERVAL`. But currently we'll return `+ MIDDLE_FREQUENCY_BUMP_INTERVAL`. Not sure this was a bug?